feature/interests/src/main/java/com/google/samples/apps/nowinandroid/feature/interests/InterestsScreen.kt

```diff
@@ -86,6 +88,8 @@ fun InterestsScreen(
     switchTab: (Int) -> Unit,
     modifier: Modifier = Modifier,
 ) {
+
+    TrackScrollJank(scrollableState = rememberLazyListState(), stateName = "interests:screen")
```

Comment:
```suggestion
    val state = rememberLazyListState()
    TrackScrollJank(scrollableState = state, stateName = "interests:screen")
```

Reply:
You need to pass the `state` to the lazyList that it refers to .. Actually what lazyList does it refer to? 🤔 
Maybe this should be part of the `InterestsContent` composable?

Reply:
Removed this again.